apps/web/src/pages/SearchRequests/SearchPage/components/SearchForm.tsx

```diff
@@ -502,8 +502,8 @@ const SearchForm = React.forwardRef<SearchFormRef, SearchFormProps>(
             })}
             text={intl.formatMessage({
               defaultMessage:
-                "If you have more detailed work location requirement, let us know in the comment section of the submission form.",
-              id: "v7sYE7",
+                "If you have more detailed work location requirement, let us know in the comment section of the submission form. You can select more than one region. If you are open to all regions, select All Regions.",
+              id: "fJ35xz",
```

Comment:
I recognize this is the copy provided but, it is confusing since there is no "All Regions" option. At first galnce (especially because of the uppercase letters) I was expecting "All Regions" to be an option in the list. Maybe we make this lower case or reword it to be clearer?

"If you are open to all regions, select all available regions"? 🤔

Reply:
Good catch. @gobyrne can you weigh in on this suggestion and if you approve of it, provide its French equivalent? I'm not sure if a @GCTC-NTGC/designers also would need to update a design somewhere (there wasn't a linked figma in the issue).

Reply:
Passing the question along to recruitment team

Reply:
@esizer I have removed the problematic copy **If you are open to all regions, select All Regions.**, please see df3c7e4616602f1cd790ca3cf7c5e63613ae3c77.